src/psyclone/transformations.py

```diff
@@ -3694,15 +3715,13 @@ def apply(self, loop, options=None):
 
         # Get a reference to the Invoke to which this loop belongs
         invoke = loop.root.invoke
-        nsm = invoke._name_space_manager
         config = Config.get().api_conf("nemo")
         index_order = config.get_index_order()
         loop_type_data = config.get_loop_type_data()
 
         loop_type = loop_type_data[index_order[outermost_dim]]
         base_name = loop_type["var"]
-        loop_var = nsm.create_name(root_name=base_name, context="PSyVars",
-                                   label=base_name)
+        loop_var = invoke.schedule.symbol_table.name_from_tag(base_name)
```

Comment:
Below this is a reference to #255 which is a closed PR. As this line simply continues the out-of-date approach could you please create another issue or fix.

Reply:
The issue here is that Nemo API still does not use a SymbolTable (this is being fixed by @arporter in #596 - so I temporally added the loop_var to the invoke symbol_table but I would not refactor the Nemo transformations as this will probably clash with the mentioned PR)

The referenced issue is indeed wrong, I will changed it to #500 and update the associated comment.